README.rst

```diff
@@ -56,26 +57,59 @@ Usage
 Extract text from HTML::
 
     >>> import html_text
-    >>> text = html_text.extract_text(u'<h1>Hey</h1>')
-    u'Hey'
+    >>> text = html_text.extract_text(u'<h1>Hello</h1> world!')
+    u'Hello world!'
+
+    >>> text = html_text.extract_text(u'<h1>Hello</h1> world!', guess_page_layout=True)
+    u'Hello
+    world!'
```

Comment:
These examples look off (actually, previous example was not good as well). I think it should look like this if you try it in a Python console:

```
>>> html_text.extract_text(u'<h1>Hello</h1> world!', guess_page_layout=True)
'Hello\n\nworld!'
```

Reply:
examples below have the same issue: it shouldn't be `text = html_text....`, just `html_text....`, otherwise there is no output